2.0/AntiqueCamera/README.md

```diff
@@ -1,13 +1,13 @@
-# Camera 
-
-## Screenshot
-
-![screenshot](screenshot/screenshot.png)
-
-## License Information
-
-Old camera model by Maximilian Kamps.  
-Dontated by UX3D for glTF testing.
-
-[![CC0](http://i.creativecommons.org/p/zero/1.0/88x31.png)](http://creativecommons.org/publicdomain/zero/1.0/)  
-To the extent possible under law, UX3D has waived all copyright and related or neighboring rights to this asset.
+# Antique Camera 
+
+## Screenshot
+
+![screenshot](screenshot/screenshot.png)
+
+## License Information
+
+Old camera model by Maximilian Kamps.  
```

Comment:
nit: Change Old to Antique to match the title?

Reply:
I was typing the word "Antique" so much that I wanted to leave some other adjective in place for this one.  I'm OK with either word here though.

Reply:
Haha. 👌